Description: Exclude indirect dependencies from mariadb_config --libs output
 On Debian systems, listing the libraries that libmariadb.so.2 is linked
 against in the output of mariadb_config --libs is inappropriate; so this
 patch removes those libraries from the output.
Author: Daniel Schepler <schepler@debian.org>
Forwarded: not-needed

--- a/mariadb_config/mariadb_config.c.in
+++ b/mariadb_config/mariadb_config.c.in
@@ -6,8 +6,7 @@
 static char *mariadb_progname;
 
 #define INCLUDE "-I@CMAKE_INSTALL_PREFIX@/@INSTALL_INCLUDEDIR@ -I@CMAKE_INSTALL_PREFIX@/@INSTALL_INCLUDEDIR@/mysql"
-#define LIBS    "-L@CMAKE_INSTALL_PREFIX@/@INSTALL_LIBDIR@/ -lmariadb "\
-                "@extra_dynamic_LDFLAGS@"
+#define LIBS    "-L@CMAKE_INSTALL_PREFIX@/@INSTALL_LIBDIR@/ -lmariadb "
 #define LIBS_SYS "@extra_dynamic_LDFLAGS@"
 #define CFLAGS  INCLUDE
 #define VERSION "@MARIADB_CLIENT_VERSION@"
